src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/HttpGoogleServicesDAOSpec.scala

```diff
@@ -241,25 +242,46 @@ class HttpGoogleServicesDAOSpec extends FlatSpec with Matchers with IntegrationT
 
   it should "crud tokens" in {
     val userInfo = UserInfo(null, null, 0, UUID.randomUUID().toString)
-    assertResult(None) { Await.result(gcsDAO.getToken(RawlsUser(userInfo)), Duration.Inf) }
-    assertResult(None) { Await.result(gcsDAO.getTokenDate(userInfo), Duration.Inf) }
+    val rawlsUser = RawlsUser(userInfo)
+    assertResult(None) { Await.result(gcsDAO.getToken(rawlsUser), Duration.Inf) }
+    assertResult(None) { Await.result(gcsDAO.getTokenDate(rawlsUser), Duration.Inf) }
     Await.result(gcsDAO.storeToken(userInfo, "testtoken"), Duration.Inf)
-    assertResult(Some("testtoken")) { Await.result(gcsDAO.getToken(RawlsUser(userInfo)), Duration.Inf) }
-    val storeTime = Await.result(gcsDAO.getTokenDate(userInfo), Duration.Inf).get
+    assertResult(Some("testtoken")) { Await.result(gcsDAO.getToken(rawlsUser), Duration.Inf) }
+    intercept[TokenResponseException] { Await.result(gcsDAO.getTokenDate(rawlsUser), Duration.Inf).get }
 
     Thread.sleep(100)
 
     val credential = gcsDAO.getBucketServiceAccountCredential
     credential.refreshToken()
     val testToken = credential.getAccessToken
     Await.result(gcsDAO.storeToken(userInfo, testToken), Duration.Inf)
-    assertResult(Some(testToken)) { Await.result(gcsDAO.getToken(RawlsUser(userInfo)), Duration.Inf) }
-    assert(Await.result(gcsDAO.getTokenDate(userInfo), Duration.Inf).get.isAfter(storeTime))
+    assertResult(Some(testToken)) { Await.result(gcsDAO.getToken(rawlsUser), Duration.Inf) }
 
-    Await.result(gcsDAO.revokeToken(RawlsUser(userInfo)), Duration.Inf)
-    Await.result(gcsDAO.deleteToken(RawlsUser(userInfo)), Duration.Inf)
-    assertResult(None) { Await.result(gcsDAO.getToken(RawlsUser(userInfo)), Duration.Inf) }
-    assertResult(None) { Await.result(gcsDAO.getTokenDate(userInfo), Duration.Inf) }
+    Await.result(gcsDAO.revokeToken(rawlsUser), Duration.Inf)
+    Await.result(gcsDAO.deleteToken(rawlsUser), Duration.Inf)
+    assertResult(None) { Await.result(gcsDAO.getToken(rawlsUser), Duration.Inf) }
+    assertResult(None) { Await.result(gcsDAO.getTokenDate(rawlsUser), Duration.Inf) }
+  }
+
+  it should "test get token date" in {
+
+    val rawlsUser = RawlsUser(RawlsUserSubjectId("110101671348597476266"), RawlsUserEmail("joel.broad.dev@gmail.com"))
+    val credential = Await.result(gcsDAO.getUserCredentials(rawlsUser), Duration.Inf).get
+
+    val refreshTokenOption = Option(credential.getRefreshToken)
```

Comment:
Because this is a test, it's OK to assume the positive case here, as it will fail with useful information if that assumption isn't true.  These would be fine:
```
val refreshToken = credential.getRefreshToken
assert(Await.result(gcsDAO.getTokenDate(rawlsUser), Duration.Inf).isValid)
```